server/routes/target-area-search.js

```diff
@@ -1,4 +1,4 @@
-const Joi = require('@hapi/joi')
+const joi = require('@hapi/joi')
```

Comment:
needs to be `const Joi` as per documentation: https://hapi.dev/family/joi/?v=16.1.7

When importing a class the variable name is capitalised.